app/models/metric/ci_mixin/capture.rb

```diff
@@ -29,9 +29,16 @@ def perf_capture(interval_name, start_time = nil, end_time = nil, target_ids = n
     raise ArgumentError, _("end_time cannot be specified if start_time is nil") if start_time.nil? && !end_time.nil?
 
     # if target (== self) is archived, skip it
-    if respond_to?(:ems_id) && ems_id.nil?
-      _log.warn("C&U collection's target is archived (no EMS associated), skipping. Target: #{log_target}")
-      return
+    if respond_to?(:ext_management_system)
+      if ext_management_system.nil?
```

Comment:
I was considering leaving this as `ems_id.nil?` but I figured if it were nil then `ext_management_system` isn't really adding a query and it reads a little better when we're using `ext_management_system.enabled?` below